### PR TITLE
To do component #13

### PR DESCRIPTION
**Pull request by RValdre**
To-Do component on dashboard done.
closes:
#13 
#21

### CODE REVIEW

File: front/src/components/TodoDialog.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        setOpen(false)
+    }
+
+    const handleSnackClose = () => {
+        setSnackOpen(false)
+    }
+
+    const handleSubmit = e => {
+        e.preventDefault()
+        if(formValue.title && formValue.description && formValue.date && formValue.assigned){
```
Comment (klillenb) [inline front/src/components/TodoDialog.js:57]: Add else statement with helper text to inform user of missing fields

**Review by klillenb — CHANGES_REQUESTED** [review]
Add else statement to adding todo, informing user of missing form values, for example as in Login.js

File: front/src/components/TodoDialog.js (hunk 1 above)
Comment (RValdre) [reply front/src/components/TodoDialog.js:57]: done

**Review by klillenb — APPROVED** [review]
Lgtm

**Pull request merged**